frontend/src/pages/listing/SellListing.tsx

```diff
@@ -0,0 +1,271 @@
+                  <Link
+                    to="/listing/newlisting"
+                    className=" flex justify-center items-center w-20 h-8  mt-4 text-sm text-gray-700 border border-stone-300 pl-3 pr-4 rounded-md hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500"
+                  >
+                    <DeleteIcon className="p-1" />
+                    Clear
+                  </Link>
+                  {id && cardId ? (
+                    <button
+                      onClick={delistItem}
+                      className=" flex justify-center items-center w-20 h-8  mt-4 text-sm text-gray-700 border border-stone-300 pl-3 pr-4 rounded-md hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500"
+                    >
+                      <VisibilityIcon className="p-1" />
+                      Delist
+                    </button>
+                  ) : null}
+                </div>
+              </div>
+              <div className="flex">
+                <div className="w-[61px] h-[23px] rounded-[10px] text-neutral-500 text-sm font-normal text-center ml-8 mt-4 border border-neutral-300">
+                  {cardInSet.set.set_code}
+                </div>
+                <div className="w-[61px] h-[23px] rounded-[10px] text-neutral-500 text-sm font-normal text-center ml-4 mt-4 border border-neutral-300">
+                  {cardInSet.rarity.rarity_code}
+                </div>
+              </div>
+              <div className="flex mt-20">
+                <div className="inline-flex items-center p-1 ml-8 border rounded-md border-gray-300">
+                  <span className="pl-1 pr-1 text-gray-700 text-sm flex items-center">
+                    <TagIcon />
+                  </span>
+                  <input
+                    type="number"
+                    name="quantity"
+                    min="0"
+                    value={formData.quantity}
+                    onChange={handleChange}
+                    className="w-24 px-2 py-0.5 text-sm text-gray-700 border-none focus:ring-0 focus:outline-none"
+                    placeholder="Quantity"
+                  />
+                </div>
+                <div className="inline-flex items-center p-1 ml-8 border rounded-md border-gray-300">
+                  <span className="pl-1 pr-1 text-gray-700 text-sm flex items-center">
+                    <DiamondIcon />
+                  </span>
+                  <select
+                    id="condition"
+                    name="condition"
+                    value={formData.condition}
+                    onChange={handleChange}
+                    className="w-28 px-2 py-0.5 text-sm text-left text-gray-700 border-none focus:ring-0 focus:outline-none"
+                  >
+                    <option value="poor">Poor</option>
+                    <option value="good">Good</option>
+                    <option value="played">Played</option>
```

Comment:
The `Good` quality is better than `Played`, so therefore, it should be after it.
`Poor -> Played -> Good -> Excellent`